apps/web-giddh/src/app/userDetails/userDetails.component.html

```diff
@@ -15,7 +15,7 @@ <h1 class="section_head pdT0">Main Auth-key</h1>
                         </div>
                         <div class="form-group clearfix mrB0">
                             <button class="btn btn-default" (click)="regenerateKey()">Regenerate Auth Key</button>
-                            <a class="btn btn-link" href="https://developers.giddh.com/collection/giddh-api-integration/46/pages/140" target="_blank">Documentation</a>
+                            <a class="btn btn-link" href="https://documenter.getpostman.com/view/117343/S1Zw8WF1?version=latest" target="_blank">Documentation</a>
```

Comment:
Please put the url in constant file and use it from there.

Reply:
@ravinderwalkover Done please check